package.json

```diff
@@ -83,6 +86,7 @@
     "web"
   ],
   "jest": {
+    "setupTestFrameworkScriptFile": "./node_modules/jest-enzyme/lib/index.js",
```

Comment:
I needed it for some reason but can't remember right now. Will try removing and see what it breaks depending on feedback on this vs #474